doc/developer-guide/api/functions/TSContSchedule.en.rst

```diff
@@ -28,31 +28,115 @@ Synopsis
 
     #include <ts/ts.h>
 
-.. function:: TSAction TSContSchedule(TSCont contp, TSHRTime timeout)
+.. function:: TSAction TSContSchedule(TSCont contp, TSHRTime timeout, TSThreadPool tp)
 
 Description
 ===========
 
-Schedules :arg:`contp` to run :arg:`delay` milliseconds in the future. This is approximate. The delay
-will be at least :arg:`delay` but possibly more. Resolutions finer than roughly 5 milliseconds will
-not be effective. :arg:`contp` is required to have a mutex, which is provided to
+Schedules :arg:`contp` to run :arg:`timeout` milliseconds in the future, on a random thread that
+belongs to :arg:`tp`. The :arg:`timeout` is an approximation, meaning it will be at least
+:arg:`timeout` milliseconds but possibly more. Resolutions finer than roughly 5 milliseconds will
+not be effective. Note that :arg:`contp` is required to have a mutex, which is provided to
 :func:`TSContCreate`.
 
+The continuation is scheduled for a particular thread selected from a group of similar threads,
+as indicated by :arg:`tp`. If :arg:`contp` already has an thread affinity set, and the thread
+type of thread affinity is the same as :arg:`tp`, then :arg:`contp` will be scheduled on the
+thread specified by thread affinity.
+
+=========================== =======================================================================================
+Pool                        Properties
+=========================== =======================================================================================
+``TS_THREAD_POOL_NET``      Transaction processing threads. Continuations on these threads must not block.
+``TS_THREAD_POOL_TASK``     Background threads. Continuations can perform blocking operations.
```

Comment:
NO_BLOCK and BLOCK as aliases or instead of NET and TASK?

Reply:
That's not a bad idea, @SolidWallOfCode  what do you think?

Reply:
A bit too radical of a change, given all the documentation and code that uses those terms.

Reply:
Adding aliases would not require changing any of that.